src/main/java/net/mcreator/ui/modgui/PlantGUI.java

```diff
@@ -1019,7 +1019,7 @@ else if (growapableType.isSelected())
 		plant.isReplaceable = isReplaceable.isSelected();
 		plant.colorOnMap = colorOnMap.getSelectedItem().toString();
 		plant.offsetType = (String) offsetType.getSelectedItem();
-		plant.aiPathNodeType = (String) aiPathNodeType.getSelectedItem();

```

Comment:
Structure GUI now has one such cast too.

Reply:
Does it? I'm not getting any warning

Reply:
Klemen has replaced `JComboBox` with `SearchableComboBox` in the latest commit so IDEA shows casting is not needed anymore.

Reply:
Ah I see, this branch wasn't up to date